docs/src/modules/components/Demo.js

```diff
@@ -354,6 +355,7 @@ class Demo extends React.Component {
           className={clsx(classes.demo, {
             [classes.demoHiddenHeader]: demoOptions.hideHeader,
           })}
+          tabIndex={-1}
```

Comment:
Make testing the tab behavior easier. Otherwise, the focus moves to the `<main>` element.